examples/screenshot_example.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import base64
+import logging
+from pathlib import Path
+
+from _helpers import get_websocket, run_and_wait_command
+
+logging.basicConfig(
+    format="%(message)s",
+    level=logging.DEBUG,
+)
+
+
+async def main():
+    # Open browser
+    websocket = await get_websocket()
+    await run_and_wait_command({
+        "id": 0,
+        "method": "session.new",
+        "params": {}
+    }, websocket)
+
+    # Open tab
+    command_result = await run_and_wait_command(
+        {
+            "id": 1000,
+            "method": "browsingContext.create",
+            "params": {
+                "type": "tab"
+            }
+        }, websocket)
+    context_id = command_result['result']['context']
+
+    # Navigate to page: https://news.ycombinator.com/
+    # To avoid network dependency in this test, use a local (static) copy.
+    pageUrl = f'file://{Path(__file__).parent.resolve()}/app.html'
+    await run_and_wait_command(
+        {
+            "id": 1001,
+            "method": "browsingContext.navigate",
+            "params": {
+                "url": pageUrl,
+                "context": context_id,
+                "wait": "complete"
+            }
+        }, websocket)
+
+    # Take screenshot
+    command_result = await run_and_wait_command(
+        {
+            "id": 1002,
+            "method": "browsingContext.captureScreenshot",
+            "params": {
+                "context": context_id,
+                "awaitPromise": True
+            }
+        }, websocket)
+
+    # `screenshot` has the base64 encoded PNG data.
+    screenshot = command_result["result"]["data"]
+
+    assert isinstance(screenshot, str)
+
+    # Save PNG file to disk.
```

Comment:
I'd comment out the file writing, and add `assert`, so that we could run the example scripts later on in CI as well. https://github.com/GoogleChromeLabs/chromium-bidi/issues/357